linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_permissions/rule.yml

```diff
@@ -67,8 +67,10 @@ ocil: |-
     is expected by the RPM database:
     <pre>$ rpm -Va | awk '{ if (substr($0,2,1)=="M") print $NF }'</pre>
 
-{{% if product == "rhel6" %}}
 warnings:
+    - general: |-
+        Profiles may require that specific files have stricter file permissions than defined by the vendor. Such files will be reported as a finding and need to be evaluated according to your policy and deployment environment.
+{{% if product == "rhel6" %}}
```

Comment:
This doesn't appear to be RHEL6 specific, and there appears to be a problem for RHEL7, possibly RHEL8. Should we remove this?

Reply:
The referenced BZ is filed against RHEL6. Are you sure about that?

Reply:
@redhatrises You mean to make the warning applicable to all products, right?

Reply:
@yuumasato yeah

Reply:
Hmm, the referenced BZ is a clone of RHEL7 bug, and the RHEL7 bug was solved: https://bugzilla.redhat.com/show_bug.cgi?id=1275532

I would still keep the warning specific for RHEL6.

Reply:
Ahhh... okay. Got it thanks for that.